app/helpers/import_rooms.rb

```diff
@@ -0,0 +1,30 @@
+require 'csv'
+require 'point'
+require 'room'
+require 'building'
+require 'polyline'
+
+rooms = %w[A B C D E F G H I]
+building = Building.create
+
+csv_text = File.read('app/assets/data/importBuilding1.csv')
+csv = CSV.parse(csv_text)
+points = {}
+rooms.each do |name|
+  polygon = {}
+  csv.each do |row|
+    next unless row[0..3].collect { |x| x.to_s[0] }.include? name.to_s
+
+    key = row[0].to_s + row[1].to_s + row[2].to_s + row[3].to_s
+    id = row[0..3].find { |e| e.to_s[0] == name.to_s }
+    number = id[1..id.length - 1] # will not work if more than 24 rooms (can be fixed with regex)
```

Comment:
Why 24? If I understand it correctly the comment could be
`# if more than 26 rooms are needed, single-char room identifier have to be replaced with e.g. multi-char ones`

Reply:
We only check  for a single-char room identifier here. Therefore the script will break if you have  multi-char identifers.
But yeah I mixed up the numbers here.
Gonna change the comment